examples/platform/nrf528xx/app/project_include/FreeRTOSConfig.h

```diff
@@ -97,7 +95,9 @@
 #define configUSE_TIMERS 1
 #define configTIMER_TASK_PRIORITY (2)
 #define configTIMER_QUEUE_LENGTH 32
-#define configTIMER_TASK_STACK_DEPTH (256)
+
+// TODO: Fix high stack usage in GenericThreadStackManagerImpl_FreeRTOS
```

Comment:
Can we file an issue here?

Reply:
There's already an issue and even another PR for reducing it 
https://github.com/project-chip/connectedhomeip/issues/2188